Cli-CredentialHelper/Installer.cs

```diff
@@ -202,11 +202,11 @@ public void DeployConsole()
             {
                 System.Security.Principal.WindowsIdentity identity = System.Security.Principal.WindowsIdentity.GetCurrent();
                 System.Security.Principal.WindowsPrincipal principal = new System.Security.Principal.WindowsPrincipal(identity);
-                if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
-                {
-                    DeployElevated();
-                    return;
-                }
+                //if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
```

Comment:
Sorry I just realized while driving the car that I'd commited this and the Version string changes, they will be reversed.